src/networks.json

```diff
@@ -893,6 +893,21 @@
     "start": 859041,
     "logo": "ipfs://QmeGbNTU2Jqwg8qLTMGW8n8HSi2VdgCncAaeGzLx6gYnD7"
   },
+  "1319": {
+    "key": "1319",
+    "name": "AITDChain Mainnet",
+    "shortName": "AITD",
+    "chainId": 1319,
+    "network": "mainnet",
+    "testnet": false,
+    "multicall": "0x16e98974D30263252aa273dC8e3d1d48380845FB",
+    "rpc": [
+      "https://node.aitd.io"
+    ],
+    "explorer": "http://aitd-explorer-new.aitd.io",
+    "start": 1000,
+    "logo": "ipfs://QmXbBMMhjTTGAGjmqMpJm3ufFrtdkfEXCFyXYgz7nnZzsy"
```

Comment:
Looks like this IPFS is not pinned @xuzuguang Use a third-party provider like pinnata to pin this :) 

https://ipfs.io/ipfs/QmXbBMMhjTTGAGjmqMpJm3ufFrtdkfEXCFyXYgz7nnZzsy